src/app/dtos/service.report.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { IsArray, IsString } from 'class-validator';
+
+/**
+ * Please note: this structure is expected to be used across all services
```

Comment:
If this object is going to be used in multiple projects, let's put it in a common library.

Reply:
I will make a separate PR once we have protocol-common in a good state.  I will not let it block getting this milestone complete however.